packages/analytics-browser/test/config/joined-config.test.ts

```diff
@@ -105,17 +117,98 @@ describe('joined-config', () => {
         };
         mockRemoteConfigFetch = {
           getRemoteConfig: jest.fn().mockResolvedValue(remoteConfig),
-          fetchTime: fetchTime,
+          metrics: metrics,
         };
         // Mock the createRemoteConfigFetch to return the mockRemoteConfigFetch
         (createRemoteConfigFetch as jest.MockedFunction<typeof createRemoteConfigFetch>).mockResolvedValue(
           mockRemoteConfigFetch,
         );
 
+        const expectedAutocapture = {
+          fileDownloads: false,
+          formInteractions: false,
+          pageViews: false,
+          attribution: false,
+          sessions: false,
+          elementInteractions: true,
+        };
+
         await generator.initialize();
         expect(generator.config.defaultTracking).toBe(true);
```

Comment:
why the `generator.config.defaultTracking` is true?

Reply:
https://github.com/amplitude/Amplitude-TypeScript/blob/e9e623419f257678acac7f70402c4683d1c8107a/packages/analytics-browser/src/config/joined-config.ts#L6-L7

Reply:
Got it `generator` is before merged.